test/multiverse/suites/rails/active_support_logger_test.rb

```diff
@@ -35,7 +35,7 @@ def test_logs_not_forwarded_by_broadcasted_logger
 
       # LogEventAggregator sees the log only once
       assert_equal 1, @aggregator.instance_variable_get(:@seen)
-      assert_equal @aggregator.instance_variable_get(:@seen_by_severity), {"DEBUG" => 1}
+      assert_equal({"DEBUG" => 1}, @aggregator.instance_variable_get(:@seen_by_severity))
```

Comment:
I tried not to touch parens when the original code did/did not have them. But rubocop really didn't like the hash rocket without the parens